tests/baselines/reference/jsdocAccessibilityTags.js

```diff
@@ -0,0 +1,225 @@
+    m() {
+        this.priv + this.prot + this.pub
+    }
+}
+class D extends C {
+    n() {
+        this.priv2 + this.prot2 + this.pub2
+    }
+}
+new A().priv + new A().prot + new A().pub
+new B().priv + new B().prot + new B().pub
+new C().priv2 + new C().prot2 + new C().pub2
+new D().priv2 + new D().prot2 + new D().pub2
+
+class Privacy {
+    /** @private */
+    constructor(c) {
+        /** @private */
+        this.c = c
+    }
+    /** @private */
+    m() {
+        return this.c
+    }
+    /** @private */
+    get p() { return this.c }
+    /** @private */
+    set p(value) { this.c = value }
+}
+
+
+//// [foo.js]
+class A {
+    constructor() {
+        /**
+         * Ap docs
+         *
+         * @private
+         */
+        this.priv = 4;
+        /**
+         * Aq docs
+         *
+         * @protected
+         */
+        this.prot = 5;
+        /**
+         * Ar docs
+         *
+         * @public
+         */
+        this.pub = 6;
+    }
+    /** @public */
+    get ack() { return this.priv; }
+    /** @private */
+    set ack(value) { }
+}
+class C {
+    constructor() {
+        /**
+         * Cp docs
+         *
+         * @private
+         */
+        this.priv2 = 1;
+        /**
+         * Cq docs
+         *
+         * @protected
+         */
+        this.prot2 = 2;
+        /**
+         * Cr docs
+         *
+         * @public
+         */
+        this.pub2 = 3;
+    }
+    h() { return this.priv2; }
+}
+class B extends A {
+    m() {
+        this.priv + this.prot + this.pub;
+    }
+}
+class D extends C {
+    n() {
+        this.priv2 + this.prot2 + this.pub2;
+    }
+}
+new A().priv + new A().prot + new A().pub;
+new B().priv + new B().prot + new B().pub;
+new C().priv2 + new C().prot2 + new C().pub2;
+new D().priv2 + new D().prot2 + new D().pub2;
+class Privacy {
+    /** @private */
+    constructor(c) {
+        /** @private */
+        this.c = c;
+    }
+    /** @private */
+    m() {
+        return this.c;
+    }
+    /** @private */
+    get p() { return this.c; }
+    /** @private */
+    set p(value) { this.c = value; }
+}
+
+
+//// [foo.d.ts]
+declare class A {
+    /**
+     * Ap docs
+     *
+     * @private
+     */
+    private priv;
+    /**
+     * Aq docs
+     *
+     * @protected
+     */
+    protected prot: number;
+    /**
+     * Ar docs
+     *
+     * @public
+     */
+    public pub: number;
+    /** @private */
+    public set ack(arg: number);
+    /** @public */
+    public get ack(): number;
+}
+declare class C {
+    /**
+     * Cp docs
+     *
+     * @private
+     */
+    private priv2;
+    /**
+     * Cq docs
+     *
+     * @protected
+     */
+    protected prot2: number;
+    /**
+     * Cr docs
+     *
+     * @public
+     */
+    public pub2: number;
+    h(): number;
+}
+declare class B extends A {
+    m(): void;
+}
+declare class D extends C {
+    n(): void;
+}
+declare class Privacy {
+    /** @private */
+    private constructor;

```

Comment:
This is a parse error in the playground. This test isn't flagging it because the input code has errors, so output validity isn't checked. You should split erroring/non-erroring cases into separate tests.

Reply:
Oh, I'd gotten so used to errors and symbol baselines co-existing I forgot that errors and declarations didn't. I'll create a separate declaration emit test.